klippy/extras/display/menu.py

```diff
@@ -608,6 +639,7 @@ def _populate(self):
 
 
 menu_items = {
+    'disabled': MenuDisabled,
```

Comment:
The config/example-menu.cfg should be updated as well.

Reply:
I'll this.